src/Core/Material/MaterialModel.hpp

```diff
@@ -30,6 +35,18 @@ class RA_CORE_API MaterialModel : public Utils::ObservableVoid
     /// DEBUG
     virtual void displayInfo() const;
 
+    virtual Utils::Color evalBSDF(Vector3f w_i, Vector3f w_o, Vector3f normal, Vector2f uv);
```

Comment:
Add documentation. Give a description of the frame in which space the computation will be done (tangent frame, world frame)